src/Module.php

```diff
@@ -63,6 +63,13 @@ class Module extends \yii\base\Module implements BootstrapInterface
      * @var LogTarget
      */
     public $logTarget;
+    
+    /**
+     *
+     * @var \yii\rbac\BaseManager|string|array the Rbac access checker [[BaseManager]] object or the application component ID of the AuthManager [[BaseManager]].
```

Comment:
```suggestion
     * @var \yii\rbac\BaseManager|string|array the RBAC access checker [[BaseManager]] object or the application 
     * component ID of the AuthManager [[BaseManager]].
     * @since 2.1.19
```